src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class MainApp {
+    public static void main(String[] args) {
+        Plane tupolev = new Plane.PlaneBuilder().setModel("TU-134")
+                .setColour("white").setMaxSpeed(850)
+                .setSize("37 meters").setType("passenger plane").build();
```

Comment:
this looks more readable in case of a pipeline (when we are calling multiple methods returning the same object in a row)
```suggestion
        Plane tupolev = new Plane.PlaneBuilder()
                .setModel("TU-134")
                .setColour("white")
                .setMaxSpeed(850)
                .setSize("37 meters")
                .setType("passenger plane")
                .build();
```